src/vue/modules/create-atomic-swap-form/components/create-atomic-swap-form.vue

```diff
@@ -313,6 +364,38 @@ export default {
       // eslint-disable-next-line max-len
       return base.CreateAtomicSwapAskRequestBuilder.createAtomicSwapAskRequest(operation)
     },
+
+    async postIssuanceOperation () {
+      const operation = base.CreateIssuanceRequestBuilder
+        .createIssuanceRequest({
+          asset: this.form.asset.code,
+          amount: this.getIssuanceAmount(),
+          receiver: this.accountBalance.id,
+          reference: this.getReference(),
+          creatorDetails: {},
+        })
+
+      await api.postOperations(operation)
+    },
+
+    getReference () {
+      return btoa(Math.random())
+    },
+
+    getIssuanceAmount () {
+      const availbaleBalance = +this.accountBalance.balance
+      const amount = +this.form.amount
+      return amount > availbaleBalance
```

Comment:
1. comparison should be done with Math.util
2. cast to number is redundant if we do the comparison with Math.util